Tools/machines/lumi-csc/submit.sh

```diff
@@ -1,16 +1,14 @@
-#!/bin/bash
+#!/bin/bash -l
 
 #SBATCH -A <project id>
-#SBATCH -J warpx
-#SBATCH -o %x-%j.out
-#SBATCH -t 00:10:00
-# Early access to the GPU partition
-#SBATCH -p eap
+#SBATCH --job-name=warpx
+#SBATCH --output=%x-%j.out
+#SBATCH --error=%x-%j.err
+#SBATCH --partition=standard-g
 #SBATCH --nodes=2
 #SBATCH --ntasks-per-node=8
-#SBATCH --cpus-per-task=1
 #SBATCH --gpus-per-node=8
-#SBATCH --gpu-bind=closest
```

Comment:
GPU bind does not work as expected?

Did you file a bug report on that in an issue number that we can track in `known issues` already?

Reply:
I am not sure... I simply followed their recommendation for the job script: https://docs.lumi-supercomputer.eu/runjobs/scheduled-jobs/lumig-job/ . They discuss binding here: https://docs.lumi-supercomputer.eu/runjobs/scheduled-jobs/distribution-binding/#gpu-binding , but they don't say that `#SBATCH --gpu-bind=closest` would not work correctly.

Reply:
I haven't tested it very recently, but a while ago `#SBATCH --gpu-bind=closest` did not work. Therefore, I also went to use their recommendation, which solved the issue.